.github/workflows/cloud-workflow.yml

```diff
@@ -0,0 +1,186 @@
+---
+name: cloud-workflow
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+    paths:
+      - 'cwf/cloud/**'
+      - 'fbinternal/cloud/**'
+      - 'feg/cloud/**'
+      - 'lte/cloud/**'
+      - 'orc8r/cloud/**'
+      - 'orc8r/lib/**'
+      - 'wifi/cloud/**'
+  pull_request:
+    branches:
+      - master
+jobs:
+  # Fail if checked-in generated code doesn't match output from
+  # generation command.
+  insync-checkin:
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - name: Run build.py
+        run: |
+            cd ${MAGMA_ROOT}/orc8r/cloud/docker
+            python3 build.py --generate
+            sudo chown -R $USER $MAGMA_ROOT/*
+            git add .
+            git status
+            git diff-index --quiet HEAD
+      - name: Notify failure to Slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "Github action insync-checkin failed"
+          SLACK_USERNAME: "Cloud workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+  # Fail if terraform variables, helm charts and orcl is out of sync
+  # Ensure all Helm chart values are in-sync across Orc8r deployment mechanisms
+  deploy-sync-checkin:
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - name: Run build.py
+        run: |
+            cd ${MAGMA_ROOT}/orc8r/cloud/deploy/orc8r_deployer/docker
+            ./run_deployer.bash --deploy-dir /tmp/deploy_dir --build --test check_all
+      - name: Notify failure to Slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "Github action deploy-sync-checkin failed"
+          SLACK_USERNAME: "Cloud workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+  # Fail if code doesn't pass formatting requirements.
+  # Upload test coverage statistics.
+  cloud-lint:
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - name: Lint cloud Go code
+        run: |
+            cd ${MAGMA_ROOT}/orc8r/cloud/docker
+            python3 build.py --lint
+      - name: Generate test coverage
+        run: |
+            cd ${MAGMA_ROOT}/orc8r/cloud/docker
+            python3 build.py --coverage
```

Comment:
Forgot to upload the lint result like here https://github.com/magma/magma/blob/5fa5cc574e7044612c951e089cbf7a46eed243cb/.circleci/config.yml#L619

Reply:
Use this github action module https://github.com/codecov/codecov-action

Reply:
Ah.. i didnt added this because i thought it can be configured in the bot.. let me add it.